src/Nethermind/Nethermind.Merge.Plugin/GC/GCKeeper.cs

```diff
@@ -150,8 +150,8 @@ private async Task ScheduleGCInternal()
         {
             // This should give time to finalize response in Engine API
             // Normally we should get block every 12s (5s on some chains)
-            // Lets say we process block in 2s, then delay 1s, then invoke GC
-            await Task.Delay(1000);
+            // Lets say we process block in 2s, then delay 500ms, then invoke GC
+            await Task.Delay(500);
```

Comment:
Can revisit later? I think it was mostly because is a gap between newPayload and forkchoice and we didn't want to GC before the forkchoice; so even 50ms or lower would probably be ok